src/assets/style.css

```diff
@@ -327,16 +327,21 @@ main h2.tip-title {
   font-weight: bold;
 }
 
-.tip img {
+.tip .tip-image {
```

Comment:
Why would this not accept the image here and render all 3 options clickable? This violates a programmers "do not repeat" rule with the images code below. Is this something specific to the libraries you are using here? Just curious.

Reply:
I have tried but it is not working so i sperated it and it worked.